examples/paths/main.go

```diff
@@ -30,33 +29,36 @@ func main() {
 
 // Main is the exported CLI executable function
 func Main() error {
-	fmt.Printf("mnemonic: %s\n", mnemonic)
+	fmt.Println("mnemonic:", mnemonic)
```

Comment:
should we replace Println with something else? logger.Debug ? or use the zap library?